backend/typescript/prisma/schema.prisma

```diff
@@ -85,17 +85,18 @@ model warning {
 }
 
 model staff {
-  id                   Int       @id @default(autoincrement())
-  roleId               Int
-  role                 role      @relation(fields: [roleId], references: [id])
+  id                   Int            @id @default(autoincrement())
+  role_id              Int
+  role                 role           @relation(fields: [role_id], references: [id])
   first_name           String
   last_name            String
-  email                String    @unique
+  email                String         @unique
   phone_number         String?
   display_name         String
   profile_picture_link String?
   tasks_assigned       task[]
   warnings_assigned    warning[]
+  notifications        notification[]
```

Comment:
unsure if this is the only thing needed to be added to the 'staff' model for the staff-notification relation